cypress/TestCases/README.md

```diff
@@ -28,3 +28,9 @@ Here are some examples of tags for different categories:
 * Tags for module: @accessibility
 * Tags for sdk: @coreSDK, @manageSDK
 * Tags for communicationModes: @sdk, @transport
```

Comment:
We should not add anything related to configModule inside FCS .  If required, that can be added in configModule readme

Reply:
Removed configModule related documentations